scala_config.bzl

```diff
@@ -39,8 +50,19 @@ def _store_config(repository_ctx):
         "ENABLE_COMPILER_DEPENDENCY_TRACKING=" + enable_compiler_dependency_tracking,
     ])
 
+    build_file_content = "\n".join([
+        'load("@bazel_skylib//rules:common_settings.bzl", "string_flag")',
+        "string_flag(",
```

Comment:
Maybe we should keep this as `string_setting` for now and do not allow to change this via command line? wdyt?

Reply:
Indeed, I see no use case for a flag at the moment.